commands/trigger.go

```diff
@@ -92,13 +93,24 @@ var triggerFireCmd = &cobra.Command{
 			return werr
 		}
 
+		if resp.StatusCode == http.StatusNoContent {
```

Comment:
I suggest starting to add go unit tests as well for such changes as it will vector toward better factoring of the code. For example a unit test that mocks an HTTP response and confirms returning the expected message and CLI code.